src/pages/ShoppingBasket/ShoppingBasket.js

```diff
@@ -0,0 +1,96 @@
+/* eslint-disable react/jsx-key */
+import React, { useEffect, useState } from 'react';
+import './ShoppingBasket.scss';
+import cartImg from '../../assets/images/shopping-cart.png';
+import { Product } from './component/Product';
+
+const ShoppingBasket = () => {
+  const [productList, setProductList] = useState([]);
+
+  const toggleSelected = value => {
+    const next = productList.map(product => {
+      if (product.productId === value) {
+        return { ...product, checkedState: !product.checkedState };
+      }
+      return product;
+    });
+
+    setProductList(next);
+  };
+
+  const temp = e => {
+    const next = productList.map(product => {
+      if (e.target.checked) return { ...product, checkedState: true };
+      else return { ...product, checkedState: false };
+    });
+    setProductList(next);
+  };
+
+  const howManyTrue = productList.filter(row => {
+    if (row.checkedState)
+      return { productId: row.productId, quantity: row.quantity };
+  });
+  const trueCount = howManyTrue.length;
+
+  useEffect(() => {
+    fetch('/data/data.json')
+      .then(res => res.json())
+      .then(data => {
+        setProductList(data);
+      });
+  }, []);
+
+  const totalSumPrice = howManyTrue.reduce(
+    (acc, cur) => acc + cur.quantity * cur.productPrice,
+    0
+  );
+
+  return (
+    <div className="ShoppingBasket">
```

Comment:
```suggestion
    <div className="shoppingBasket">
```
컴포넌트와 같은 이름으로 잘 구현해 주셨는데 소문자로 시작해주세요!